packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_SequencerFeeVault.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+
+/* Library Imports */
+import { Lib_PredeployAddresses } from "../../libraries/constants/Lib_PredeployAddresses.sol";
+
+/* Contract Imports */
+import { OVM_ETH } from "../predeploys/OVM_ETH.sol";
+
+/**
+ * @title OVM_SequencerFeeVault
+ * @dev Simple holding contract for fees paid to the Sequencer. Likely to be replaced in the future
+ * but "good enough for now".
+ *
+ * Compiler used: optimistic-solc
+ * Runtime target: OVM
+ */
+contract OVM_SequencerFeeVault {
+
+    /*************
+     * Constants *
+     *************/
+
+    // Minimum ETH balance that can be withdrawn in a single withdrawal.
+    uint256 public constant MIN_WITHDRAWAL_AMOUNT = 15 ether;
```

Comment:
we should be able to update this value once we have L2 upgradeability regenesis-ed in, right?

Reply:
Correct